microsoft_teams/plugin.spec.yaml

```diff
@@ -201,7 +201,7 @@ types:
        title: Words
        description: The message split by spaces into a list of words. This list makes finding and using parameters in chat commands easier
        type: "[]string"
-       required: true
+       required: false
```

Comment:
This should still be required - it should just be an empty list if there are no elements

Reply:
That will make this a major version bump. Just FYI

Reply:
We can identify how many people are using this 2.3.0 and if none - we don't have to make them update. Would like to avoid a major bump if we can.

Reply:
We are going with false. Will fix in a future ticket.

Reply:
See SOAR-3226